Makefile

```
@@ -8,8 +8,9 @@ run:
 
 .PHONY: test
 test:
+	poetry run mypy .
+	poetry run flake8 .
```

better to run this before the test suite so we fail fast